easybuild/easyconfigs/r/RELION/RELION-2.1_cuda_capabilities.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur relion-2.1.orig/CMakeLists.txt relion-2.1/CMakeLists.txt
+--- relion-2.1.orig/CMakeLists.txt	2017-12-15 13:55:13.000000000 +0100
++++ relion-2.1/CMakeLists.txt	2018-06-01 11:20:17.908580307 +0200
+@@ -75,7 +75,7 @@
+     set(CUDARCH "-arch=sm_35")
+ else(NOT DEFINED CUDA_ARCH)
+     message(STATUS "Using provided CUDA_ARCH=${CUDA_ARCH}")
+-    set(CUDARCH "-arch=sm_${CUDA_ARCH}")
++    set(CUDARCH ${CUDA_ARCH})
```

Comment:
this is a bit weird? the else condition is `NOT DEFINED CUDA_ARCH` and then you just use it?

Reply:
Nah, the (NO DEFINED...) is a comment referring to the actual if-stmt

Reply:
yes. I agree it's confusing if you see this for the first time